libs/membership/api/src/lib/periodic-job/periodic-job.service.ts

```diff
@@ -198,7 +201,8 @@ export class PeriodicJobService {
         // Don't send custom mails for deactivated subscriptions
         deactivation: {
           is: null
-        }
+        },
+        confirmed: true
```

Comment:
Don't we want custom mails to be sent for non-confirmed Subscriptions? And just deactivate charging?